pkg/manifest/os_test.go

```diff
@@ -369,3 +370,63 @@ func TestHostnameDoesNotIncludeHostnameStage(t *testing.T) {
 	st := manifest.FindStage("org.osbuild.hostname", pipeline.Stages)
 	require.Nil(t, st)
 }
+
+func TestAddFilesOS(t *testing.T) {
+	os := manifest.NewTestOS()
```

Comment:
This is (almost) identical to the OS test is there potential for code sharing here?

Reply:
Made two new functions, one for creating the test files and another for collecting the copy stage destination paths.